packages/debug-test/CDPDebugger.ts

```diff
@@ -0,0 +1,400 @@
+import * as lodash from 'lodash';
```

Comment:
Add `@format` in this file and others so that they are formatted, once ESLint is properly configured.

Reply:
Added '@format' to all *.ts files (together with copyright message); auto-formatted files.